prow/config.yaml

```diff
@@ -387,6 +389,10 @@ presubmits:
           value: /etc/ssh-key-secret/ssh-private
         - name: JENKINS_GCE_SSH_PUBLIC_KEY_FILE
           value: /etc/ssh-key-secret/ssh-public
+        # TODO(bentheelder): this probably isn't the most idiomatic location,
```

Comment:
comment is no longer applicable?

Reply:
`# TODO(benthelder): clean up stale comments`

Reply:
Done.